src/tools/rbd_mirror/InstanceReplayer.cc

```diff
@@ -334,6 +334,8 @@ void InstanceReplayer<I>::start_image_replayer(
 
   std::string global_image_id = image_replayer->get_global_image_id();
   if (!image_replayer->is_stopped()) {
+    dout(10) << "image replayer has not yet stopped for global_image_id: "
```

Comment:
Again, stick to the `variable=value` convention:
```suggestion
    dout(10) << "image replayer is not stopped for global_image_id="
```